dev_readme.md

```diff
@@ -37,7 +85,18 @@
 * Root level: this is the level of the project which contains all the others as files and subfolders
 * Terminal: this is an application on your computer which allows you to use code to modify your computer's software and files. 
```

Comment:
I love that you define these terms!

Reply:
Oh, I can't take credit for this. It was all @GeorgiaHCA !! 👏  👏  👏